The_simplest_classes_and_objects/src/com/androsor/prog_class/student/Student.java

```diff
@@ -0,0 +1,66 @@
+package com.androsor.prog_class.student;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class Student {
+
+    private String fullName;
+    private int group;
+    private int[] progress;
+
+    public Student(String fullName, int group, int[] progress) {
+        this.fullName = fullName;
+        this.group = group;
+        this.progress = progress;
+    }
+
+    public String getFullName() {
+        return fullName;
+    }
+
+    public int getGroup() {
+        return group;
+    }
+
+    public int[] getProgress() {
+        return progress;
+    }
+
+    public void setFullName(String fullName) {
+        this.fullName = fullName;
+    }
+
+    public void setGroup(int group) {
+        this.group = group;
+    }
+
+    public void setProgress(int[] progress) {
+        this.progress = progress;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Minor: I do believe that you could use any util methods even from external libraries.

Reply:
The same for other methods.